Panel.h

```diff
@@ -67,10 +68,10 @@ struct Panel_ {
    Vector* items;
    int selected;
    int oldSelected;
-   int selectedLen;
+   size_t selectedLen;
    void* eventHandlerState;
    int scrollV;
```

Comment:
For consistency, this would need to change to `size_t` too.

Reply:
I was considering that changing the `scrollH` type to `size_t` would be a bad move. It should have been `unsigned int` instead. It doesn't make sense to allow the UI to display 2 GiB of characters nor allow user to scroll past 2 GiB characters. In other words, it might be better to cap the `scrollH` value to `INT_MAX` whenever we have such a large data.

Reply:
Capping the internal value is independent from the data type used (to some extend).

Also, the marked line is about `scrollV`, not `scrollH`.

Reply:
What I was considering is setting the data type of `scrollH` to `unsigned int`. And for consistency change `scrollV` to `unsigned int` as well. Even though I consider the `scrollV` changes be separate commits (because it's not directly related to or depend on `RichString`).